File: domain/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LabMPP.domain
{
    class Account:Entity<String>
    {
        public long idUser { get; set; }
        public string password { get; set; }

        public Account(string username, long idUser, string password):base(username)
        {
            this.idUser = idUser;
            this.password = password;
        }

        public string getUsername()
        {
            return base.id;
        }

        public override string ToString()
        {
            return getUsername();
        }
    }
}

File: src/main/java/domain/Account.java
package domain;

public class Account extends Entity<String>{
    private Long idUser;
    private String password;

    public Account(String username, Long idUser , String password) {
        super(username);
        this.idUser=idUser;
        this.password = password;
    }

    public Long getIdUser() {
        return idUser;
    }

    public String getPassword() {
        return password;
    }

    public String getUsername(){
        return super.getId();
    }

    @Override
    public String toString() {
        return getUsername();
    }
}
